src/gamecontroller.cs

```diff
@@ -3044,4 +3248,104 @@ interface UISceneConnectionOptions_GameController {
+
+	[TV (17, 4), Mac (14, 3), iOS (17, 4), MacCatalyst (17, 4)]
+	[Protocol (BackwardsCompatibleCodeGeneration = false)]
+	interface GCAxis2DInput {
+		[Abstract]
+		[NullAllowed, Export ("valueDidChangeHandler", ArgumentSemantic.Copy)]
+		GCAxis2DInputValueDidChangeCallback ValueDidChangeHandler { get; set; }
+
+		[Abstract]
+		[Export ("value")]
+		GCPoint2 Value { get; }
+
+		[Abstract]
+		[Export ("analog")]
+		bool Analog { [Bind ("isAnalog")] get; }
+
+		[Abstract]
+		[Export ("canWrap")]
+		bool CanWrap { get; }
+
+		[Abstract]
+		[Export ("lastValueTimestamp")]
+		double LastValueTimestamp { get; }
+
+		[Abstract]
+		[Export ("lastValueLatency")]
+		double LastValueLatency { get; }
+
+		[Abstract]
+		[Export ("sources", ArgumentSemantic.Copy)]
+		NSSet<IGCPhysicalInputSource> Sources { get; }
+	}
+
+	interface IGCAxis2DInput { }
+
+	[TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[Protocol (BackwardsCompatibleCodeGeneration = false)]
+	interface GCPhysicalInputSource {
+		[Abstract]
+		[Export ("elementAliases", ArgumentSemantic.Copy)]
+		NSSet WeakElementAliases { get; }
+
+		// FIXME: strongly typed bindings for ElementAliases
+		// @required @property (readonly, copy) NSSet<NSString<GCPhysicalInputElementName> *> * _Nonnull elementAliases;
+
+		[Abstract]
+		[NullAllowed, Export ("elementLocalizedName")]
+		string ElementLocalizedName { get; }
+
+		[Abstract]
+		[NullAllowed, Export ("sfSymbolsName")]
+		string SfSymbolsName { get; }
+
+		[Abstract]
+		[Export ("direction")]
+		GCPhysicalInputSourceDirection Direction { get; }
+	}
+
+	interface IGCPhysicalInputSource { }
+
+	[TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (GCControllerInputState))]
+	[DisableDefaultCtor]
+	interface GCControllerLiveInput : GCDevicePhysicalInput {
+		[NullAllowed, Export ("unmappedInput")]
+		GCControllerLiveInput UnmappedInput { get; }
+
+		// 'new' because this is also implemented in GCDevicePhysicalInput, but with a less defined property type (IGCDevicePhysicalInputState)
+		[Export ("capture")]
+		new GCControllerInputState Capture { get; }
+
+		[NullAllowed, Export ("nextInputState")]
+		// The property type is both GCControllerInputState + implements the GCDevicePhysicalInputStateDiff protocol,
+		// which can't be expressed in C#. Choosing to bind as GCControllerInputState.
+		// 'new' because this is also implemented in GCDevicePhysicalInput, but with a less defined property type (NSObject)
+		new GCControllerInputState NextInputState { get; }
+	}
+
+	[TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface GCControllerInputState : GCDevicePhysicalInputState {
+	}
+}
+
+namespace Foundation {
+	using GameController;
+
+	partial interface NSValue {
+		[TV (17, 4), Mac (14, 3), iOS (17, 4), MacCatalyst (17, 4)]
+		[Static]
+		[Export ("valueWithGCPoint2:")]
+		NSValue FromCGPoint2 (GCPoint2 point);
```

Comment:
Typo? `CGPoint2` vs `GCPoint2`
```suggestion
		NSValue FromGCPoint2 (GCPoint2 point);
```